modules/mono/editor/GodotTools/GodotTools/Build/BuildManager.cs

```diff
@@ -273,6 +285,56 @@ private static bool PublishProjectBlocking(BuildInfo buildInfo)
             return success;
         }
 
+        private static async Task<bool> BuildProjectAsync(BuildInfo buildInfo)
+        {
+            if (!File.Exists(buildInfo.Project))
+                return true; // No project to build.
+
+            if (_buildInProgress != null)
+            {
+                ShowBuildErrorDialog("Build already in progress!");
+                return false; // Already a build in progress.
+            }
+
+            bool success = await BuildAsync(buildInfo);
+
+            if (!success)
+            {
+                ShowBuildErrorDialog("Failed to build project");
+            }
+            else
+            {
+                Internal.EditorPlugin_PopupToast(".Net Build Completed.");
```

Comment:
To follow the branding guidelines, use `.NET`.

```suggestion
                Internal.EditorPlugin_PopupToast(".NET Build Completed.");
```